### PR TITLE
Cleanup ci and setup.py

### PR DESCRIPTION
**Pull request by jettify**
* cleanup yaml file syntax
* update setup.py to follow best practices
* enable pyroma on flake check

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -52,13 +52,30 @@ def read_version():
...
+    'Operating System :: POSIX',
+    'Development Status :: 4 - Beta'
+    'Framework :: AsyncIO',
+]
+
+
 setup(
     name="create-aio-app",
     version=read_version(),
-    description="Mykhailo Havelia",
```
Comment (Arfey) [inline setup.py]: 😁 short and clear

**Comment by Arfey** [comment]
So, i add `long_description` for pypi and we can make new version.

**Comment by jettify** [comment]
We usually do something like this
https://github.com/aio-libs/aiozipkin/blob/e2bd3558e4c80b98a306525a8c6d31318f0c3b61/setup.py#L51
but readme here is md not rst ...

**Comment by asvetlov** [comment]
Do you insist on markdown?

**Comment by webknjaz** [comment]
markdown is dumb, it was invented to save n00bs from learning very basic html, rst has better set of features :)

**Pull request merged**